crypto/curve25519/curve25519.c

```diff
@@ -2091,7 +2096,12 @@ static void x25519_scalar_mult(uint8_t out[32], const uint8_t scalar[32],
 }
 
 void X25519_keypair(uint8_t out_public_value[32], uint8_t out_private_key[32]) {
-  RAND_bytes(out_private_key, 32);
+  if (!RAND_bytes(out_private_key, 32)) {
+    // This is a public void function and can't be updated
+    OPENSSL_cleanse(out_public_value, 32);
+    OPENSSL_cleanse(out_private_key, 64);
```

Comment:
```suggestion
    OPENSSL_cleanse(out_private_key, 32);
```

Reply:
This is a very good catch. This also shows we need a callback + asan/valgrind flavor.